src/sentry/incidents/models.py

```diff
@@ -141,7 +141,7 @@ def duration(self):
 class IncidentSnapshot(Model):
     __core__ = True
 
-    incident = models.OneToOneField("sentry.Incident")
+    incident = models.OneToOneField("sentry.Incident", on_delete=models.CASCADE)
```

Comment:
> The on_delete argument for ForeignKey and OneToOneField is now required in models and migrations.

We don't use OneToOneField a lot, but it might be good for consistency's sake to do the same thing you did with flexible fk.